gcp/appengine/frontend3/src/styles.scss

```diff
@@ -1252,7 +1262,10 @@ dl.vulnerability-details,
       .icon {
         background: #fff;
         border-radius: 999px;
-        color: #d46d59;
+        --md-icon-button-icon-color: #d46d59;
+        --md-icon-button-hover-icon-color: #d46d59;
+        --md-icon-button-focus-icon-color: #d46d59;
+        --md-icon-button-pressed-icon-color: #d46d59;
```

Comment:
♻️ The same as above comment on system colours.

Reply:
updated [c7a9bea](https://github.com/google/osv.dev/pull/2314/commits/c7a9bea483be59f447aaaf8fe27bf0051712f093)